synapse/handlers/room_member.py

```diff
@@ -702,6 +709,9 @@ def do_3pid_invite(
                     Codes.FORBIDDEN,
                 )
 
+        # Check whether we'll be ratelimited
+        yield self.base_handler.ratelimit(requester, update=False)
```

Comment:
Where do you update the ratelimit counter?

Reply:
It happens when we send the invite event :)

Reply:
Aaah that's the part that's already ratelimited, gotcha, thanks.